pkg/stream/server_frame.go

```diff
@@ -600,9 +600,16 @@ func (c *Client) handleConsumerUpdate(readProtocol *ReaderProtocol, r *bufio.Rea
 		return
 	}
 	consumer.setPromotedAsActive(isActive == 1)
+
+	// shouldn't this only be run if the consumer is active?
```

Comment:
you are right; you can remove the comment. maybe add a comment with more information. thank you

Reply:
I have removed the comment for now